widget_driver/doc/testing.md

```diff
@@ -88,15 +88,15 @@ void main() {
 ### Create a driver
 
 To create your driver you will need a helper function.  
-This is because the `Driver` needs a build context as a parameter to its constructor.
+This is because the `Driver` might need a BuildContext as a parameter to its `didUpdateBuildContext`.
```

Comment:
This sentence is somehow wrong now. Well it is correct that didUpdateBuildContext needs the buildcontext, but your driver does not need it any more, so I am not sure if this whole paragraph is still valid (saying that you _need_ a helper function for the Tester). I guess it is still handy to have a helperfunction but the buildcontext is taken out of this equation now from my point of view.